lib/parse/data/parseAudioData.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { IRssItem } from '@interfaces/data';
+import parseAudioData from './parseAudioData';
+
+describe('lib/parse/data', () => {
+  describe('parseAudioData', () => {
+    const mockRssItem: IRssItem = {
```

Comment:
I get that the interface is coupled to the `RssParser` types, so mocking them here makes sense...

But my gut feel here is, I kind of want a test that involves parsing an XML string?  Is that crazy?  Maybe that belongs more in a `fetchRssFeed.spec.ts`, or even a slightly more integration-y layer that mocks the HTTP endpoint with an RSS string.  (I'd be fine to punt on this as well - would be an interesting task for someone new to the project to tackle as well).

Reply:
We don't currently have a use case where we accept raw XML from the params, and since the RSS fetch/parser handles the conversion of the XML to JS object structure (I haven't written anything in this code that parses the XML directly), I felt like writing a test that uses an XML string would be testing the RSS parser rather than the code I wrote. Since the RSS parser has different methods for parsing XML strings and parsing RSS returned from a URL, a different parsing method than what would be used in the project would be use in the tests.

Reply:
I was thinking something more like `nock`.  But yeah, it would be more of an integration-y test, and I'm fine punting on that until if/when it would provide some value for the project.